apps/frontend/src/components/ui/ProfileImage.tsx

```diff
@@ -8,6 +8,8 @@ const goudyBookletter = Goudy_Bookletter_1911({
 });
 
 export const getProfileBackgroundColor = (user: UserData) => {
+  const isHalloween: boolean = false;
```

Comment:
Didn't wind up doing this, because there was some kind of annoying passthrough on black, but not other colors. In theory I think this solution would work, which is why I kept it up -- we can quickly add new colors if need be.